test/puppeteer/docs.test.ts

```diff
@@ -1,11 +1,63 @@
 // Imports
 import path from 'path';
 
+// Destructured constant for readability
+const { keyboard } = page;
+
+// Helper functions used in multiple tests
+const menuDisplayState = () => page.evaluate(() => document.querySelector<HTMLDivElement>('#menu').style.display);
+const currentFocusId = () => page.evaluate(() => document.activeElement.id);
+
 // Tests
-beforeAll(async () => {
+beforeEach(async () => {
 	await page.goto(`file://${path.join(__dirname, '..', '..', 'docs', 'build', 'index.html')}`);
+	await page.waitFor('#menubutton');
```

Comment:
Is it necessary to wait for this to appear?

Reply:
It does not seem to be pertinent after the latest changes.